spark/sql/dataframe.go

```diff
@@ -146,6 +146,19 @@ type DataFrame interface {
 	Repartition(ctx context.Context, numPartitions int, columns []string) (DataFrame, error)
 	// RepartitionByRange re-partitions a data frame by range partition.
 	RepartitionByRange(ctx context.Context, numPartitions int, columns ...column.Convertible) (DataFrame, error)
+	// Replace Returns a new DataFrame` replacing a value with another value.
+	// Values toReplace and Values must have the same type and can only be numerics, booleans,
+	// or strings. Value can have None. When replacing, the new value will be cast
+	// to the type of the existing column.
+	//
+	// For numeric replacements all values to be replaced should have unique
+	// floating point representation.
+	Replace(ctx context.Context, toReplace []types.PrimitiveTypeLiteral,
+		values []types.PrimitiveTypeLiteral) (DataFrame, error)
+	// ReplaceWithColumns is an alias for Replace but allows to specify a subset of columns that
+	// should be used for the replacement.
+	ReplaceWithColumns(ctx context.Context, toReplace []types.PrimitiveTypeLiteral,
+		values []types.PrimitiveTypeLiteral, cols []string) (DataFrame, error)
```

Comment:
if we allow to use empty `cols`, then maybe we can use `cols ...string` instead, so people can simply omit the value?

Reply:
will do